<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LeistungstraegerBox</class>
 <widget class="QGroupBox" name="LeistungstraegerBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>467</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>670</width>
    <height>470</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="title">
   <string>GroupBox</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="leistungstrGerLabel">
       <property name="text">
        <string>Leistungsträger:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="leistungstrGerLineEdit">
       <property name="minimumSize">
        <size>
         <width>526</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>526</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="vornameLabel">
       <property name="text">
        <string>Vorname:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="vornameLineEdit"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="nachnameLabel">
       <property name="text">
        <string>Nachname:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="nachnameLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="straELabel">
       <property name="text">
        <string>Straße:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="straELineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="pLZLabel">
       <property name="text">
        <string>PLZ:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="pLZLineEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="ortLabel">
       <property name="text">
        <string>Ort:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="ortLineEdit"/>
     </item>
     <item row="4" column="1">
      <widget class="Line" name="line">
       <property name="lineWidth">
        <number>3</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="telefonLabel">
       <property name="text">
        <string>Telefon:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="telefonLineEdit"/>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="faxLabel">
       <property name="text">
        <string>Fax:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="faxLineEdit"/>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="eMailLabel">
       <property name="text">
        <string>E-Mail</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLineEdit" name="eMailLineEdit"/>
     </item>
     <item row="11" column="1">
      <widget class="Line" name="line_2">
       <property name="lineWidth">
        <number>3</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Anmerkung:</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QTextEdit" name="textEdit"/>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Ansprechpartner:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
